common/quotas/policy.go

```diff
@@ -83,7 +80,8 @@ func (d *DomainRateLimitPolicy) Allow(domain string) bool {
 
 	if !ok {
 		// create a new limiter
-		domainLimiter := rate.NewLimiter(rate.Limit(domainRps), domainRps)
+		initialRps := d.domainRPS()
+		domainLimiter := NewRateLimiter(&initialRps, _defaultRPSTTL, int(d.domainRPS()))
```

Comment:
should we consider allowing temporary burst?

Reply:
Current config allows the burst to take up the full limit, but we do not actually expose a get N tokens API, so the burst really isnt useful at this time. Its more like a placeholder.. But with more complicated policy it will come into play..

Reply:
it does not has to be related to `consume N token` for some heavily lifting APIs.
my point is allow customer to temporary use more tokens, that is what token bucket for.

Reply:
Ah yes, that was an oversight. Updated..